cores/esp8266/core_esp8266_postmortem.cpp

```diff
@@ -110,10 +110,27 @@ static void cut_here() {
     ets_putc('\n');
 }
 
-void __wrap_system_restart_local() {
-    register uint32_t sp asm("a1");
-    uint32_t sp_dump = sp;
-
+static void postmortem_report(uint32_t sp_dump) __attribute__((used));
```

Comment:
Is it dropped, somehow?
I don't see any errors on my side after removing this and adding `static` below

Reply:
Updated. There is some scenario where I need this, but not here. :confused: